src/components/List.js

```diff
@@ -79,6 +80,21 @@ const List = ({ token }) => {
     return daysUntilNextPurchase > 0 ? daysUntilNextPurchase : 0;
   };
 
+  // Confirm that the user would like to delete this item
+  const confirmDelete = (item) => {
+    const confirmed = window.confirm(
+      'Are you sure you want to delete this item?',
+    );
+    if (confirmed) {
+      try {
+        db.collection(token).doc(item.id).delete();
+      } catch (error) {
+        setDeleteError(`Problem deleting ${item.itemName}`);
```

Comment:
We currently dont have a way of testing getting into the `catch` block. If anyone has any suggestions of how to do that (so the error message shows up when there is an actual error deleting the item) we would appreciate that.

Reply:
I wonder if you put a fake token in the `try` block if it would error:

`db.colleciont('fake').doc(item.id).delete();`

Reply:
That's a great idea!

Reply:
Alternatively, if you ever want to trigger a `catch` block just to smoke test your code, you can always just put `throw new Error("Some random error message")` in the `try` block and that will automatically send the code straight to the `catch` block.